assignments/array-methods.js

```diff
@@ -56,28 +56,63 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+
+runners.forEach(runner => {
+    fullName.push(`${runner.first_name} ${runner.last_name}`);
+});
+
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+
+runners.map((runner, index) => {
+    allCaps[index] = runner.first_name.toUpperCase();
```

Comment:
Good job looking at the docs and seeing that you can use multiple parameters to set index.  Also don't forget that map returns an array so you can also take that value and assign it to `allCaps`